models/lindeberg2022simple-ontorights/metadata.yaml

```diff
@@ -1,13 +1,14 @@
-title: Simple OntoRights
+title: Easy OntoRights
 acronym:
 issued: 2022
-modified:
+modified: 2023
```

Comment:
Shouldn't this be 2024? Given that the latest paper has not even been published yet?

Reply:
I used the publication date provided on the [ontology's landing page](https://joranl.github.io/human-rights-ontology/ontorights-access.html), as I believe it is the most reliable date, reflecting when it was truly made available to users.